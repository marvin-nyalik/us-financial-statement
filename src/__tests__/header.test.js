import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Header from '../components/header';

describe('Test the header component', () => {
  it('Matches the snapshot', () => {
    const { container } = render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
