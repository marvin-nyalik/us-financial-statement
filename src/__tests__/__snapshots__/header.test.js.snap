// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the header component Matches the snapshot 1`] = `
<div>
  <div
    class="header"
  >
    <p>
      <a
        class="year"
        href="/"
      >
        <i
          class="bx bx-chevron-left-circle"
        />
         
      </a>
    </p>
    <p
      class="heading"
    />
    <div
      class="icons"
    >
      <i
        class="bx bxs-microphone-alt"
      />
      <i
        class="bx bx-cog"
      />
    </div>
  </div>
</div>
`;
